common/test/acceptance/tests/studio/test_studio_settings_certificates.py

```diff
@@ -52,7 +52,7 @@ def create_and_verify_certificate(self, course_title_override, existing_certs, s
             certificate.signatories[idx].name = signatory['name']
             certificate.signatories[idx].title = signatory['title']
             certificate.signatories[idx].organization = signatory['organization']
-            certificate.signatories[idx].upload_signature_image('Signature-{}.png'.format(idx))
+            certificate.signatories[idx].upload_signature_image('Signature_{}.png'.format(idx))
```

Comment:
Yeah you can use one of the old files to invalid scenario i.e `Signature-0.png` other one of the existing file can be resized.